paymentintent.go

```diff
@@ -56,6 +56,15 @@ const (
 	PaymentIntentOffSessionRecurring PaymentIntentOffSession = "recurring"
 )
 
+// PaymentIntentSetupFutureUsage is the list of allowed values for SetupFutureUsage.
+type PaymentIntentSetupFutureUsage string
+
+// List of values that PaymentIntentSetupFutureUsage can take.
+const (
+	PaymentIntentSetupFutureUsageOffSession PaymentIntentSetupFutureUsage = "off_session"
+	PaymentIntentSetupFutureUsageOnSession  PaymentIntentSetupFutureUsage = "on_session"
```

Comment:
I'm sure you're well aware, but: It looks like this is still correct according to current docs, but I thought we'd decided to rename in the end.

Reply:
Yes, I thought so too but after all we did not. They gavel-ed the original proposal!

Reply:
Ah, weird. SG though.